src/encoded/static/components/viz/QuickInfoBar.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import React from 'react';
+import React, { useMemo } from 'react';
```

Comment:
Codacy found an issue: ['useMemo' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6178047)